apps/docs/content/guides/api/using-custom-schemas.mdx

```diff
@@ -19,6 +19,12 @@ ALTER DEFAULT PRIVILEGES FOR ROLE postgres IN SCHEMA myschema GRANT ALL ON ROUTI
 ALTER DEFAULT PRIVILEGES FOR ROLE postgres IN SCHEMA myschema GRANT ALL ON SEQUENCES TO anon, authenticated, service_role;
 ```
 
+<Admonition type="note">
+
+It is necessary to run these commands after adding new tables to the schema.
```

Comment:
@mansueli, just want to double-check what is meant by adding new tables to the schema? My first interpretation would be every time you `create table schema.whatever` you need to rerun all the SQL commands, but wouldn't the `alter default privileges` already take care of this? Or am I misunderstanding either/both of those statements?